### PR TITLE
Fix namespacing

### PR DESCRIPTION
**Pull request by YingboMa**
Requires https://github.com/SciML/ModelingToolkit.jl/pull/2249

### CODE REVIEW

**Review by isaacsas — APPROVED** [review]
Thanks! Feel free to bump this once the MTK PR gets merged.

File: Project.toml
Comment (isaacsas) [inline Project.toml:33]: Should this be 8.66?

Comment (YingboMa) [reply Project.toml:33]: Ah, you are right.

**Comment by isaacsas** [comment]
Thanks! I'll merge and release a new version once the doc build finishes ok.

**Comment by YingboMa** [comment]
Great! Thanks!

**Pull request merged**
